Gemfile.lock

```diff
@@ -441,4 +441,4 @@ RUBY VERSION
    ruby 3.0.0p0
 
 BUNDLED WITH
-   2.3.11
+   2.3.15
```

Comment:
No, not really, that’s just my bundler version :)

Reply:
I guess we wouldn't want whoever runs bundler to update this file, right?

Reply:
It’s intended this way:

* When you run bundler and bundler updates the lock file, it might introduce changes that aren’t compatible with older versions of bundler.
* When someone using an older version of bundler then tries to run bundler, they can get an error message if there are compatibility issues.
* Also, the version information helps bundler to update the lock file once you have a newer bundler version installed.

Reply:
It is not my experience that it is "intended this way" at all. It is my experience that if someone on the team has a newer version of bundler and wants to update it in the Germfile.lock that a separate PR would be opened for it. I have just recently opened a PR and run `bundle install` and it has downgraded it in the `Germfile.lock`, there were no error messages in that case. See https://github.com/tactilenews/100eyes/pull/1416/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR454

Reply:
To avoid downgrading the bundler version, I've run:

```sh
gem update bundler
bundle update --bundler
bundle install
```
on another PR and now, I have also introduced a bump to the bundler version. See https://github.com/tactilenews/100eyes/pull/1412/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR474

This is how you are accustomed to handling this workflow? Whoever has the higher version updates the `Gemfile.lock` and others play catch-up?